### DISCUSSION TITLE
Version 1.11.1

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#1791] [Changelog]
## Bugfixes
- #1741 disable-auto-save disables manual save. (prio-2)
- #1753 Userstyle only addons are not listed as running. (prio-4)
- #1767 dark-www scrollbars doesn't change color on hover. (prio-4)
- #1770 variable-manager lists broadcasts. (prio-3)
- #1784 onion-skinning is incompatible with folders. (prio-3)
- #1786 block-palette-icons has a typo in addon name. (prio-4)
- #1786 variable-manager has a typo. (prio-4)
- #1803 mute-project makes sound blocks stuck. (prio-3)
- #1803 mute-project breaks when going in and out of editor. (prio-4)
- #1812 folders does not handle empty-name sprites inside folders properly. (prio-4)
- #1815 longer-wiwo logs error when on other people's profiles. (prio-4)
- #1826 forum-search is extremely slow. (prio-3)
- #1827 exact-count logs error when scratchstats is enabled. (prio-4)
- #1849 editor-dark-mode makes scrolling code area impossible. (prio-3)
- #1851 animated-thumb setting name is confusing. (prio-3)
- #1852 editor-devtools does not remove "clean up blocks" from context menu. (prio-4)

## Changes
- #1778 Portuguese translation is moved to Portuguese Brazil. (prio-4)
- #1799 Video sensing option in load-extensions is removed. (prio-2)
- #1847 Feedback button in variable-manager is removed. (prio-4)

**Comment by Mr-Programmer-777** [comment]
A bit late, don't you think? 😅

**Reply by Mr-Programmer-777** [reply]
> A bit late, don't you think? sweat_smile
Nevermind, figured it was 1.11.1, not 1.11

**Comment by mxmou** [comment]
> #1767 dark-www does not make scrollbars dark. (prio-4)

It did make them dark, but they didn't change color on hover.

**Comment by WorldLanguages** [comment]
Releasing tomorrow, but need 2 things sorted out:
1. @TheColaber wants to suggest an alternative that would not require #1782, with PR #1799
2. #1741 being merged is a coup :P (and addons shouldn't change behavior in patch releases unless there's a clear bug, and this is not the case)

**Reply by WorldLanguages** [reply]
3. Do we want more variable manager feedback? We had enough already

**Comment by Hans5958** [comment]
Could anyone check #1846 ASAP? @WorldLanguages @apple502j
